### DISCUSSION TITLE
Welcoming Brendan Chou to Commonware

### DISCUSSION THREAD

**Discussion by patrick-ogrady** [commonwarexyz/monorepo#305] [Announcements]
Thrilled to welcome [@B_Chou](https://x.com/B_Chou) to Commonware!

Brendan, previously First Employee and Head of Engineering at dYdX, is the only engineer I've ever met with firsthand experience scaling a popular application from ETH smart contract to custom L2 all the way to standalone L1.

Having this one-of-a-kind perspective at the table as we develop Commonware's first primitives is essential: ensuring we ship things that actually provide value while minimizing the friction that drives great builders away down the road.

Hiring Brendan was a no brainer. Also a no brainer? Let's chat soon 🤝

https://x.com/B_Chou/status/1869410700558467396

**Comment by JackLamCHN** [comment]
🎉This is such an exciting announcement! We’re thrilled to welcome [@B_Chou](https://x.com/B_Chou)!

dYdX is a powerful cross-chain anonymous DEX. Does this mean that Commonware will also have similarly, or even more, robust on-chain functionalities like cross-chain exchanges and encrypted communications in the future?

Can't wait for this moment to arrive.

**Reply by patrick-ogrady** [reply]
> Does this mean that Commonware will also have similarly, or even more, robust on-chain functionalities like cross-chain exchanges and encrypted communications in the future?

Our primary focus is enabling great developers to push the envelope of what is possible onchain (whether it be a DEX or something new). I thought hiring Brendan was a great fit not to build a new dYdX but because he had hands-on experience pushing the envelope for the past 7 years or so (not that it hurts to have DEX experience in-house if we go that route 😉 ).

**Reply by JackLamCHN** [reply]
Yes, let's bring together outstanding developers to create a powerful Commonware that can empower existing DApps. If all this comes to fruition, I believe a Commonware with strong cross-chain trading capabilities will be a game changer. Let's look forward to its arrival!

🎉 By the way, 2025 is here—wishing everyone a Happy New Year! A new journey for Commonware begins!

**Comment by adizere** [comment]
While on the Comet team, I've interacted with Brendan and was impressed, great hire!
